src/docs/apidoc.js

```diff
@@ -0,0 +1,29 @@
+const options = {
+  definition: {
+    openapi: '3.0.0',
+    info: {
+      title: 'API Library',
+      version: 1.0,
+      description: 'Swagger Api Documentation',
+    },
+  },
+  servers: [
+    {
+      url: 'http://localhost:3000/',
+      description: 'Api server',
+    },
```

Comment:
Since this link is dynamic according to deployments, would keep it in the env file and have the URL conditional. if the environment variable of the URL exists assign it if it does not keep the localhost with that port.